test/transmuxer/transmuxer_integration.js

```diff
@@ -275,6 +275,13 @@ describe('Transmuxer Player', () => {
     });
 
     it('H.265 in TS', async () => {
+      const chromeVersion = shaka.util.Platform.chromeVersion();
+      if (shaka.util.Platform.isWindows() &&
+          chromeVersion && chromeVersion === 117) {
+        // It appears that Chrome 117 beta in Windows is incorrectly reporting
```

Comment:
This sounds like something that should be put into `shaka.polyfill.MediaSource`. Unless this only comes up in the tests.

Reply:
We are already do the same in some integration tests. If the problem persist in 2-3 versions more, I'll do it.

Reply:
Fair enough.